### PR TITLE
Removing specific beta4 build and fixing assembly loading issue on vnext

### PR DESCRIPTION
**Pull request by awec**
Prevents error for people who don't have 11566.

### CODE REVIEW

**Comment by awec** [comment]
Potential dumb question: is the CI failing for dnx builds expected?

**Comment by khellang** [comment]
Probably. I didn't even know we had a DNX branch. I guess the CI build hasn't been updated to be able to build for DNX.

**Comment by horsdal** [comment]
We do (obviously) but it is very much a WIP. Let's ping @jchannon on this one and get it merged or closed.

File: src/global.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿{
     "projects": [ "src" ],
     "sdk": {
-        "version": "1.0.0-beta4-11566"
+        "version": "1.0.0-beta4"
```
Comment (jchannon) [inline src/global.json]: Is it worth adding a `-*`

Comment (awec) [inline src/global.json]: not sure if that makes a difference.

File: src/Nancy/Bootstrapper/NancyBootstrapperBase.cs [hunk 2]
```diff
@@ -245,6 +245,8 @@ public void Initialise()
 
             this.ConfigureApplicationContainer(this.ApplicationContainer);
 
+            AppDomainAssemblyTypeScanner.UpdateTypes();
```
Comment (jchannon) [inline src/Nancy/Bootstrapper/NancyBootstrapperBase.cs]: Not sure about this one so bat signal for @NancyFx/most-valued-minions  @NancyFx/owners

Comment (phillip-haydon) [inline src/Nancy/Bootstrapper/NancyBootstrapperBase.cs]: Its fine. It fixed a bug in master we found last night. Decided its fine to include cos its only run on start up.

/approved.

**Comment by jchannon** [comment]
Thanks!!

**Pull request merged**